src/Conductor/Controllers/DefinitionController.cs

```diff
@@ -24,9 +24,21 @@ public DefinitionController(IDefinitionService service)
 
         [HttpGet]
         [Authorize(Policy = Policies.Author)]
-        public ActionResult<IEnumerable<string>> Get()
+        public ActionResult<IEnumerable<Definition>> Get([FromQuery] PaginationParameter parameter)
         {
-            return new string[] { "value1", "value2" };
+            var result = parameter.IsValid() ?
+
+                _service.GetDefinitions()
+                    .OrderByDescending(x => x.Version)
+                    .DistinctBy(p => p.Id)
```

Comment:
We should not group by this, we should return all versions.  If the desire to to retrieve the latest version, we should design an API specific to that use case rather than hijack the generic API for it.

Reply:
This is more complicated that it seemed at first sight :P.

Now I'm thinking that returning paginated, all version definitions, may not be the most useful thing to do. 

Imagine the case where we would return 10 entries which would consist of: 
- Definition1 with all of its 7 versions
- Definition2 with only 3 versions out of 20

Do you think there is any useful scenario for this way of returning definitions?

In my scenario, I need a list of unique definitions to display them visually in a list and let the user decide which one to edit. The simplest approach would be to edit the latest version.

What about returning unique definitions and have another endpoint return all the versions of a specific definition based on its Id?

Sorry for the long talk.

Reply:
From a Restful point of view, I think doing a `GET` on `/definitions` should return all, but we could add some query parameters to filter / group them by workflowID... perhaps something like OData or something similar?

Reply:
Let me think about a simple Filter/Group solution. My original intention was to really get the filtered result :P.